### PR TITLE
Changes made to the buttons

### PR DESCRIPTION
**Pull request by Tesluach**
Let me hope you will like

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,7 @@
       color: #000;
       font-weight: 200;
       box-shadow: 2px 10px 5px 0 #000;
+      font-family: 'Bershire Swash Regular';
```
Comment (iamTes) [inline index.html:67]: Congrats! This font-family looks awesome

File: index.html [hunk 2]
```diff
@@ -92,6 +93,16 @@
       font-size: 2rem;
       font-family: 'Fascinate Inline Black';
     }
+
+    #btn1:hover {
+      background: #038803;
```
Comment (iamTes) [inline index.html]: I would love if you change this color to something that is nice looking

File: index.html [hunk 3]
```diff
@@ -92,6 +93,16 @@
       font-size: 2rem;
       font-family: 'Fascinate Inline Black';
     }
+
+    #btn1:hover {
+      background: #86c232;
```
Comment (Tesluach) [inline index.html:98]: How does it look so far.

Comment (iamTes) [reply index.html:98]: I love that. Thanks

**Comment by iamTes** [comment]
I have liked this work. Congrats

**Pull request merged**